packages/manager/modules/telecom-spare/src/phones/order/phones-order.html

```diff
@@ -0,0 +1,212 @@
+<div class="telecom-spare-phone-order-section">
+    <header>
+        <tuc-section-back-link data-tuc-section-back-link-to-state="spare.phones"
+            data-tuc-section-back-link-title="{{ ::'phones_order_back_link' | translate }}">
+        </tuc-section-back-link>
+        <h1 data-translate="phones_order_new_phone_title"></h1>
```

Comment:
As mentionned earlier, use a `oui-back-button` to harmonize

Reply:
already done, not push yet

Reply:
Ok